src/services/testUtil.ts

```diff
@@ -0,0 +1,41 @@
+import client from '../client';
+import {JobState} from '../proto/pps/pps_pb';
+
+export const waitForJobSuccess = async (
+  jobId: string,
+  pipelineName: string,
+  interval = 500,
+  timeout = 30000,
+) => {
+  return new Promise((resolve, reject) => {
+    const pachClient = client({ssl: false, pachdAddress: 'localhost:30650'});
+    const pps = pachClient.pps();
+    const start = Date.now();
+    const poll = () =>
+      setTimeout(async () => {
+        const job = await pps.inspectJob({
```

Comment:
Does the `wait` option for `inspectJob` not work for this scenario?

Reply:
looks like it does! did not know that was a thing good catch